### DISCUSSION TITLE
About Perplexity Data

### DISCUSSION THREAD

**Discussion by rahuljploft** [perplexityai/api-discussion#50] [Q&A]
I am using Perplexity to fetch events by location name and date. But  I got inaccurate data. Can you please let me know when i hit same prompt hit on preplexity UI I got accurate data and another side when i pass same prompt in my code (API) I got different data.

**Comment by shubhang98** [comment]
Please refer to our FAQs here that talks about why you could be seeing differences - https://docs.perplexity.ai/faq/faq#why-are-the-results-from-the-api-different-from-the-ui. 

Another thing is the API does not support location today - passing in location in the prompt might work but might not be accurate all the time.
